README.md

```diff
@@ -24,76 +24,276 @@ undirected graphs, and electrical circuits.
 
 ### Getting Started
 
-###### **MENTION!** 
-This does not work with Safari as a browser, please test it with a different browser for the moment (Chrome and Firefox work)
+Before you can start visualizing your models inside a web browser, you need to
+install the following Node package using NPM (Node Package Manager): browser-sync.
+Install it in the root directory of the project.
 
+	npm install browser-sync
 
-Before you can start visualizing your models inside a web browser, you need to
-install the following Node packages using NPM (Node Package Manager): browser-sync.
+You can start the server for watching your file using this command and sourcing your **.mc** file (this would be if your file is in the root directory of the project):
+
+	node src/visual/boot.js path/to/your_file.mc
+
+This will prompt you to the port on your localhost on which the server is started, now if you modify and save the file which contains your models, it should generate a file called **data-source.js** and reflect the update in the browser immediately. The generated file will appear in the src/visual/webpage directory.
 
-	npm install  browser-sync
+## Data types
 
+This environment supports the datatypes of type _model_. Right now this includes:
+* DFA
+* NFA
+* Directed graph
+* Graph
+* Binary tree
 
-This is how you would write your DFA:
+## This is how you would write your DFA:
 
 Starting State:
 
 	let startState = X_i
 
-States (integers):
+States:
```

Comment:
It is better to include a real example that runs. This code include pseudo code, e.g. `...` which is not real code. Please use a real example and then explain each of the lines in the example.  Now you have one file `test/test.mc` that includes a lot. It would be cleaner if you separated these examples into different files, e.g. `dfa.mc`, `digraph.mc` etc.

Reply:
This has been updated in the README, so they are actual values. The test.mc file has also been divided into smaller examples.